modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -350,12 +359,11 @@ public Response muteDevice(@HeaderParam("Accept") String acceptHeader,
 		}
 	}
 
-	//TODO : Need to properly construct ConfigOperation
 	@POST
 	@Produces(MediaType.APPLICATION_JSON)
 	@Path("install-application")
```

Comment:
better if the path name can be changed to a noun instead of a verb (i.e. install-application) -  as REST APIs normally represent resources